app/javascript/pages/country/widget/widgets/widget-tree-gain/widget-tree-gain.js

```diff
@@ -40,7 +63,7 @@ class WidgetTreeGainContainer extends PureComponent {
   }
 
   getSentence = () => {
```

Comment:
See comment above about putting in the selectors. I think it is better as then we only pass the data from mapStateToProps once and dont have to reassign in the container.